src/cljs/replumb/repl.cljs

```diff
@@ -827,7 +853,14 @@
   file is found). It is mutually exclusive with :load-fn! and will be
   ignored in case both are present
 
-  * :src-paths - a vector of paths containing source files
+  * `:write-file-fn!` a synchronous 2-arity function that accepts a file-path
```

Comment:
In `repl.cljs` I don't put usually backticks as we are not including it in `codox`